src/core/assets/copy/coreCopy.ts

```diff
@@ -23,6 +23,10 @@ export const coreCopy = {
     inputNumberMax: {
         max: 'Max',
     },
+    date: {
+        ago: 'ago',
+        left: 'left',
+    },
```

Comment:
Strings are currently defined per component, I would define these on the modules copy for the proposalDataListItem component and discuss if we want to introduce shared copies later on

Reply:
✅